### PR TITLE
[mstable] Add mStable strategy

### PR DESCRIPTION
**Pull request by chrisjgf**
Create strategy for voting with stkMTA & stkBPT balances

### CODE REVIEW

**Review by alsco77 — APPROVED** [review]

File: src/strategies/mstable/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# mStable
```
Comment (ChaituVR) [inline src/strategies/mstable/README.md:1]:
```suggestion
# mstable
```

File: src/strategies/mstable/examples.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "name": "mStable",
```
Comment (ChaituVR) [inline src/strategies/mstable/examples.json:3]:
```suggestion
    "name": "mstable",
```

File: src/strategies/mstable/index.ts [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+const abi = [
+  {
+    inputs: [{ internalType: 'address', name: 'account', type: 'address' }],
+    name: 'getVotes',
+    outputs: [{ internalType: 'uint256', name: '', type: 'uint256' }],
+    stateMutability: 'view',
+    type: 'function'
+  }
+];
```
Comment (ChaituVR) [inline src/strategies/mstable/index.ts:15]:
```suggestion
const abi = [ 'function getVotes(address account) view returns (uint256)' ];
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Comment by ChaituVR** [comment]
Strategy is live: https://snapshot.org/#/strategy/mstable

**Pull request merged**